sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -51,7 +53,20 @@ describe("EventHub Receiver", function(): void {
     await producerClient.close();
   });
 
-  afterEach("close the receiver link", async function(): Promise<void> {
+  beforeEach(async () => {
+    debug("Creating the clients..");
```

Comment:
Any reason we need a debug statement here? Were you having timing issues or something interesting?

Reply:
Nope, that is copy/paste.
BeforeEach sections in other files had this. So, I made the assumption that either you or @chradek added them for a purpose, so I re-used it to keep the same pattern.

Reply:
Oh, I did not add them for any purpose :smile:

Reply:
For all I know I added them. And...I also did not add them for any purpose :)

Reply:
great, I am surrounded by things with no purpose

![](https://media1.giphy.com/media/hmpqVzwPARTmo/giphy.gif)

Reply:
Removed the debug statements in https://github.com/Azure/azure-sdk-for-js/pull/9521/commits/2f443bca0bd7b96beb4206955ff140a247ddfd58